src/Auth/Models/Throttle.php

```diff
@@ -81,6 +81,11 @@ public function addLoginAttempt()
 
         if ($this->getLoginAttempts() >= static::$attemptLimit) {
             $this->suspend();
+
+            /*
+             * Fire the user 'onSuspend' event
+             */
+            $this->user()->onSuspend();
```

Comment:
This would have to be `$this->getUser()` or `$this->user`, `$this->user()` refers to a BelongsTo relationship instance which passes method calls off to a QueryBuilder.